tests/baselines/reference/regularExpressionGroupNameUnicodeEscapes.errors.txt

```diff
@@ -0,0 +1,299 @@
+                                                                 
+!!! error TS1514: Expected a capturing group name.
+    /(?<_🌚>)(?<_\u{1F31A}>)(?<_\uD83C\uDF1A>)\k<_🌚>\k<_\u{1F31A}>\k<_\uD83C\uDF1A>/;
+         
+!!! error TS1005: '>' expected.
+                ~
+!!! error TS1515: Named capturing groups with the same name must be mutually exclusive to each other.
+                 
+!!! error TS1005: '>' expected.
+                               ~
+!!! error TS1515: Named capturing groups with the same name must be mutually exclusive to each other.
+                                
+!!! error TS1005: '>' expected.
+                                                  
+!!! error TS1005: '>' expected.
+                                                         
+!!! error TS1005: '>' expected.
+                                                                       
+!!! error TS1005: '>' expected.
+    
+    // Lone leading surrogate
+    /(?<\uD800>)(?<\u{D800}>)\k<\uD800>\k<\u{D800}>/;
+        
+!!! error TS1514: Expected a capturing group name.
+                   
+!!! error TS1514: Expected a capturing group name.
+                                
+!!! error TS1514: Expected a capturing group name.
+                                          
+!!! error TS1514: Expected a capturing group name.
+    /(?<_\uD800>)(?<_\u{D800}>)\k<_\uD800>\k<_\u{D800}>/;
+         
+!!! error TS1005: '>' expected.
+                    ~
+!!! error TS1515: Named capturing groups with the same name must be mutually exclusive to each other.
+                     
+!!! error TS1005: '>' expected.
+                                   
+!!! error TS1005: '>' expected.
+                                              
+!!! error TS1005: '>' expected.
+    
+    // Lone trailing surrogate
+    /(?<\uDFFF>)(?<\u{DFFF}>)\k<\uDFFF>\k<\u{DFFF}>/;
+        
+!!! error TS1514: Expected a capturing group name.
+                   
+!!! error TS1514: Expected a capturing group name.
+                                
+!!! error TS1514: Expected a capturing group name.
+                                          
+!!! error TS1514: Expected a capturing group name.
+    /(?<_\uDFFF>)(?<_\u{DFFF}>)\k<_\uDFFF>\k<_\u{DFFF}>/;
+         
+!!! error TS1005: '>' expected.
+                    ~
+!!! error TS1515: Named capturing groups with the same name must be mutually exclusive to each other.
+                     
+!!! error TS1005: '>' expected.
+                                   
+!!! error TS1005: '>' expected.
+                                              
+!!! error TS1005: '>' expected.
+    
+    // Fake surrogate pair with extended Unicode escapes – invalid even if the intended character has both properties
+    /(?<\u{D800}\u{DC00}>)\k<\u{D800}\u{DC00}>/;
+        
+!!! error TS1514: Expected a capturing group name.
+                ~~~~~~~~
+!!! error TS1538: Unicode escape sequences are only available when the Unicode (u) flag or the Unicode Sets (v) flag is set.

```

Comment:
The first escape sequence error is not reported as it starts at the same position as the TS1514 error. Is it really desirable? Personally, I would prefer to have an additional check that the last error is not of length zero, in which case it is not considered to be overlapping.